models/cis.cattle.io.clusterscan.js

```diff
@@ -0,0 +1,40 @@
+import { CIS } from '@/config/types';
+import { downloadFile } from '@/utils/download';
+import { colorForState } from '@/plugins/steve/resource-instance';
+
+export default {
+  availableActions() {
+    const out = this._standardActions;
+    const downloadLogs = {
+      action:     'downloadReport',
+      enabled:    true,
+      icon:       'icon icon-fw icon-chevron-right',
+      label:      'Download Report',
+      total:      1,
+    };
+
+    out.push(downloadLogs);
```

Comment:
Custom actions on a resource usually go at the top (unshift) instead of the bottom.